docs/source/configuration/authorization.mdx

```diff
@@ -1,71 +1,135 @@
+Whether your graph is small or large, you may need to restrict sensitive information to authenticated users or enforce conditional access rules. It's more efficient to do this on the supergraph level, rather than in your underlying services.
+The Apollo Router lets you **control access to specific fields and types across your supergraph** through the `@authenticated` and `@requiresScopes` directives:
 
-They are defined as follows:
+- The `@authenticated` directive works in a binary way: authenticated requests can access the specified field or type and unauthenticated requests can't.
+- The `@requiresScopes` directive allows granular access control through scopes you define.
 
-```graphql
-directive @authenticated on OBJECT | FIELD_DEFINITION | INTERFACE | SCALAR | ENUM
+You define and use these directives on subgraph schemas, and GraphOS [composes](#composition-and-federation) them on to the supergraph schema.
+The router then enforces these directives on all incoming requests.
 
-directive @requiresScopes(scopes: [String!]!) on OBJECT | FIELD_DEFINITION | INTERFACE | SCALAR | ENUM
+## Prequisites
+
+To use the router's authorization directives, you need to either configure [JWT authentication](./authn-jwt) or add a [router service coprocessor](../customizations/coprocessor) that adds **claims** to a request's context.
+**Claims** are the individual details of a request's scope. They might include details like the ID of the associated user and/or any scopes assigned to that user.
+
+If you configure [JWT authentication](./authn-jwt), the Apollo Router automatically adds a JWT token's claims to the request's context at the `apollo_authentication::JWT::claims` key.
+To participate in the authorization process, the coprocessor you add needs to set the key `apollo_authentication::JWT::claims` in request contexts.
+
+### JWT authentication configuration
+
+To-do: Recipe for JWT auth with these directives
+
+### Coprocessors for authorization
+
+To-do: Coprocessors specifications
+
+## Authorization directives
+
+### `@authenticated`
+
+The `@authenticated` directive marks specific fields and types as requiring authentication.
+It works by checking for the`apollo_authentication::JWT::claims` key in a request's context.
+If the request is authenticated, the router executes the query in its entirety.
+
+For unauthenticated requests, the router removes `@authenticated` fields before planning the query.
+The router filters out fields that require authentication and only executes the parts of the query that don't require it.
+If every field in a particular subgraph's query is marked as requiring authentication, this can eliminate entire subgraph requests.
+
+#### Prequisites
+
+To use the `@authenticated` directive in a subgraph you can either:
+- [import it from the `@link` directive](/federation/federated-types/federated-directives/#importing-directives) like so:
+
+```graphql
```

Comment:
Is this now accurate? I'm unsure when users can use `@link` vs. having to define the directive.